shell/platform/windows/public/flutter_windows.h

```diff
@@ -21,8 +21,8 @@ extern "C" {
 typedef void (*VoidCallback)(void* /* user data */);
 
 // Opaque reference to a Flutter window controller.
-typedef struct FlutterDesktopViewControllerState*
-    FlutterDesktopViewControllerRef;
+struct FlutterDesktopViewController;
+typedef struct FlutterDesktopViewController* FlutterDesktopViewControllerRef;
```

Comment:
I feel like this isn't needed. We can still have a `struct FlutterDesktopViewControllerState` that wraps the view controller.